cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -191,16 +244,16 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
           resultTable()
         }
 
-        p("Like other `SHOW` commands, the output can also be processed using `YIELD` / `WHERE` / `RETURN`.")
+        p("Also here can the output be processed using `YIELD` / `WHERE` / `RETURN`.")
```

Comment:
```suggestion
        p("The output be processed using `YIELD` / `WHERE` / `RETURN` here as well.")
```

Reply:
It feels like you are missing a word `The output be processed ...` does not sound right to me

Reply:
You're absolutely right, there should be a _can_ in there.